src/plant/Flower.java

```diff
@@ -1,4 +1,16 @@
 package plant;
 
+import java.lang.reflect.Field;
+
 public class Flower {
+
```

Comment:
Where are the fields of this class? And what do you think about making this class abstract?